src/NUnitEngine/mock-assembly/MockAssembly.cs

```diff
@@ -103,6 +103,7 @@ public class MockAssembly
             public const int Inconclusive = MockTestFixture.Inconclusive;
 
 #if !NETSTANDARD1_6
+            //Replace with TestContext.CurrentContext.TestDirectory ?
```

Comment:
You can't do that since it's initialized in static constructor. This needs to be as it is because the field is accessed externally, by our tests, eons after the test was run. You can just drop the comment or add one that says we can't use TestContext.TestDirectory here.